plugins/dev-tools/src/testblocks/index.js

```diff
@@ -24,15 +24,17 @@ import {category as styleCategory, onInit as initStyle} from './style';
 /**
  * The Test blocks toolbox.
  */
-export const toolbox = [
-  alignCategory,
-  basicCategory,
-  connectionsCategory,
-  dragCategory,
-  fieldsCategory,
-  mutatorsCategory,
-  styleCategory,
-];
+export const toolbox = {
+  "contents": [
```

Comment:
nit: use single quote instead of double.

Reply:
done